platform/ios/src/MGLMapView.h

```diff
@@ -922,6 +929,18 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)flyToCamera:(MGLMapCamera *)camera withDuration:(NSTimeInterval)duration peakAltitude:(CLLocationDistance)peakAltitude completionHandler:(nullable void (^)(void))completion;
 
+/**
+ Moves the viewpoint to a different location without using a transition.
+ 
+ Transition animations can be defined by other classes and use this method to
+ update the map.
+ 
+ @param camera The new viewpoint.
+ @param padding The minimum padding (in screen points) that would be visible
+ 
+ */
+- (void)jumpToCamera:(MGLMapCamera *)camera edgePadding:(UIEdgeInsets)insets;
```

Comment:
My understanding is that this essentially exposes core's `jumpTo` method. But it feels close to `-flyToCamera:withDuration:completionHandler:` with a 0 duration (which calls down to the internal method `-_flyToCamera:edgePadding:withDuration:peakAltitude:completionHandler:`.

I think this could be confusing. Thoughts?

Secondly, I think this method might need a completion handler.

Reply:
I definitely need direct access to the edgePadding argument. I also don't want any animation since I'm controlling the animation outside of the iOS SDK. I looked into using one of the setCamera methods but I couldn't get it to work the way I wanted. Plus it appeared to drop frames vs this method. This could probably use a closer look.

Reply:
Right know we have two direct flavors to set the camera the `setCamera: XXX` and `flyToCamera`, `jumpToCamera` sounds like falls into the first flavor, would it be possible to add `setCamera:edgePadding:` instead?